examples/aurelia-kitchen-sink/.storybook/preview.ts

```diff
@@ -0,0 +1,9 @@
+import { addParameters, addDecorator } from '@storybook/aurelia';
+
+addParameters({
+  options: {
+    brandTitle: 'Aurelia Kitchen Sink',
+    brandUrl: 'https://github.com/storybookjs/storybook/tree/master/examples/aurelia-kitchen-sink',
```

Comment:
These should be set in `addons.setConfig` in `manager.js`

Reply:
Moved